dds_cli/data_remover.py

```diff
@@ -128,12 +130,14 @@ def remove_all(self, *_, **__):
         )
 
         # Print out response - deleted or not?
-        if "removed" not in response_json:
+        if "removed" not in response_json and "message" not in response_json:
             raise dds_cli.exceptions.APIError(
                 "Malformatted response detected when attempting "
                 f"to remove all files from {self.project}."
             )
 
+        dds_cli.utils.console.print("\n" + response_json["message"])
+
```

Comment:
I suggest changing it to this. If any of these are missing it's malformed, not if both are missing. 
```suggestion
        # Print out response - deleted or not?
        queue_message: str = response_json.get("message")
        if "removed" not in response_json or not queue_message:
            raise dds_cli.exceptions.APIError(
                "Malformatted response detected when attempting "
                f"to remove all files from {self.project}."
            )
        
        dds_cli.utils.console.print("\n" + queue_message)

```